spec/exmples/contacts_accessor_spec.rb

```diff
@@ -0,0 +1,65 @@
+require File.expand_path(File.dirname(__FILE__) + '../../spec_helper')
+
+describe GoogleContactsApi::ContactsAccessor do
+    before(:all) do
+      @contact_json_hash = contact_json_hash
+      @contact = GoogleContactsApi::Contact.new(@contact_json_hash)
+    end
+
+    let(:api) { double("api") }
+    let(:test_class) {
+      Class.new do
+        include GoogleContactsApi::ContactsAccessor
+        def initialize(api)
+          @api = api
+        end
+      end
+    }
+    describe ".get_contacts" do
+      it "should get the contacts using the internal @api object" do
+        expect(api).to receive(:get).with("contacts/default/full", kind_of(Hash)).and_return(Hashie::Mash.new({
+          "body" => "some response",
+          "code" => 200
+        }))
+        allow(GoogleContactsApi::ContactSet).to receive(:new).and_return("contact set")
```

Comment:
I don't understand the point of stubbing out `api.get` if you aren't checking that `ContactSet.new` is receiving those stubbed attributes?